src/preloadjs/LoadQueue.js

```diff
@@ -1222,14 +1222,14 @@ this.createjs = this.createjs || {};
 	 * @method getItems
 	 * @param {Boolean} loaded Determines if only items that have been loaded should be returned. If false, in-progress
 	 * and failed load items will also be included.
-	 * @returns {Array} A list of objects that have been loaded. Each item includes the {{#crossLink "LoadItem"}}{{/crossLink}},
+	 * @returns {Object[]} A list of objects that have been loaded. Each item includes the {{#crossLink "LoadItem"}}{{/crossLink}},
```

Comment:
The yuidoc documentation doesn't seem to specify what is valid types (at least can't find it). 
My editor doesn't complain about that format, but that doesn't mean much.

How can you see if it is correct?